scripts/qsscheck.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import argparse
+import fnmatch
+import os
+import os.path
+import re
+import sys
+import tinycss.css21
+import PyQt5.QtWidgets
+
+
+def get_skins(mixxx_path):
+    skins_path = os.path.join(mixxx_path, 'res', 'skins')
+    for entry in os.scandir(skins_path):
+        if entry.is_dir():
+            yield entry.name
+
+
+def make_glob(name):
+    return re.sub(r'<.*>', '*', name)
+
+
+def get_global_names(mixxx_path):
+    classnames = set()
+    objectnames = set()
+    for root, dirs, fnames in os.walk(os.path.join(mixxx_path, 'src')):
+        for fname in fnames:
+            ext = os.path.splitext(fname)[1]
+            if ext in ('.h', '.cpp'):
+                fpath = os.path.join(root, fname)
+                with open(fpath, mode='r') as f:
+                    for line in f:
+                        classnames.update(set(
+                            re.findall(r'^\s*class\s+([\w_]+)', line)))
```

Comment:
Can you factor out all the regexes into top-level global constants and then refer to them by name where they are used? I see that they are only used once, but that pattern makes it easier to understand what each regex is doing and why it's being used